### ISSUE TITLE
Ability to curate files with same header information.

### ISSUE THREAD

**Issue by royaljust** [clearlydefined/website#722]
Some components use the same header on multiple files.  

For example, RocksDB uses the following header on over 100 files.
// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under both the GPLv2 (found in the
//  COPYING file in the root directory) and Apache 2.0 License
//  (found in the LICENSE.Apache file in the root directory).

Being able to group files by license header so that they might be curated in a single shot would provide substantial improvement in curation time.

**Comment by storrisi** [comment]
Probably a combination of filter by License and multi-select on files would help?

**Comment by royaljust** [comment]
Thanks @storrisi - I think that those features certainly would help curation, but we'd still need to verify that the text is the same for each file.

I understand from @pombredanne that the header data is being harvested, so we might be able to sort by header or apply curations across identical. I'd prefer to trust computers to determine whether 100+ header files are identical than a person :)

**Comment by storrisi** [comment]
@dabutvin i could be wrong but the service is not currently sending an "header" field for each file. Could you confirm it?

**Comment by dabutvin** [comment]
that's right, but we may be able to pull that data out from the scancode results

**Comment by storrisi** [comment]
If we consider this implementation really useful, we can address the right implementation on the service and then do the same on the website. Thoughts?
